# QA
`make updated` failed with:

```
Processing: logs/api/2/api.jujucharms.com.log-20170105.anon.gz
Traceback (most recent call last):
  File "long-running/longrunning.py", line 390, in <module>
    cli()
  File "/home/bac/git/gypsy-danger/.venv/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/bac/git/gypsy-danger/.venv/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/bac/git/gypsy-danger/.venv/lib/python3.5/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/bac/git/gypsy-danger/.venv/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/bac/git/gypsy-danger/.venv/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "long-running/longrunning.py", line 331, in updatedb
    load_logfiles()
  File "long-running/longrunning.py", line 86, in load_logfiles
    ''', ['/'.join([path_parts[-2], logname]),])
sqlite3.InterfaceError: Error binding parameter 0 - probably unsupported type.
Makefile:74: recipe for target 'updatedb' failed
```

I ran it again and it succeeded.

--- a/long-running/longrunning.py
+++ b/long-running/longrunning.py
@@ -83,7 +83,7 @@ def load_logfiles():
             res = c.execute('''
                 SELECT * FROM loaded_logs
                 WHERE logfile = ?
-            ''', ['/'.join([path_parts[-2], logname]),])
+            ''', ['/'.join([path_parts[-2], logname]), ])
             found = res.fetchone()
 
             if found:

